CODEOWNERS

```diff
@@ -973,6 +973,7 @@ tests/components/trace/* @home-assistant/core
 homeassistant/components/tractive/* @Danielhiversen @zhulik @bieniu
 tests/components/tractive/* @Danielhiversen @zhulik @bieniu
 homeassistant/components/trafikverket_train/* @endor-force @gjohansson-ST
+tests/components/trafikverket_train/* @endor-force @gjohansson-ST
```

Comment:
I think this has leaked in from another PR.
```suggestion
```

Reply:
Apparently yes. No idea why it's doing this but I guess it will continue until I finish that other PR :(